index.rst

```diff
@@ -201,7 +206,7 @@
                <h2>Upcoming workshops</h2>
 
                <ul>
-               <li>2024-12-08: Mastering Geospatial Visualizations with GMT/PyGMT at AGU Fall Meeting 2024.</li>
+
                </ul>
```

Comment:
Lines 208-210 can be removed.

Reply:
Removed in commit https://github.com/GenericMappingTools/website/pull/160/commits/d0a8d07d820a3bc1d12cd4bada5dba9ca1702d08.